docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+
+  php:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+      args:
+        - PHP_VERSION=${PHP_VERSION:-7.1}
+        - XDEBUG=${XDEBUG:-0}
+    volumes:
+      - ./:/docker
+      - ./docker/php.ini:/usr/local/etc/php/php.ini
+    links:
+      - mongo
```

Comment:
This `links` section is [deprecated](https://docs.docker.com/compose/compose-file/#links), and is no longer needed in recent versions of docker-compose.

Reply:
Version 3:
```
MongoConnectionException: No suitable servers found (`serverSelectionTryOnce` set): [Failed to resolve 'mongo']
```

I've changed to v 3.7 to try again

Reply:
3.7
```
1) DoctrineMongoODMModuleTest\Doctrine\PaginationAdapterTest::testItemCount
MongoConnectionException: No suitable servers found (`serverSelectionTryOnce` set): [Failed to resolve 'mongo']
```

Reply:
Staying with links for this PR.  PRs for user networks for this are welcome.

Reply:
That's weird, it should not be necessary, at least that's what I understand from the docs: https://docs.docker.com/compose/networking , anyway, not a big deal.

Reply:
What I read says "Use a network" but I've never seen a docker-compose network config (which is why I'd like a PR).

Reply:
From what I read, it's supposed to be created automatically. I'm going to try locally, but I use Fedora, which means no `docker`, and no `docker-compose` since the latest version, but `podman` and `podman-compose`, which are supposed to be drop-in replacements. Wish me luck

Reply:
Doesn't work… guess I'll try a PR then.

Reply:
Here you go: https://github.com/TomHAnderson/DoctrineMongoODMModule/pull/1